.github/workflows/main.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# GitHub recommends pinning actions to a commit SHA.
+# To get a newer version, you will need to update the SHA.
+# You can also reference a tag or branch, but the action may change without warning.
+
+name: Newsletter issue
```

Comment:
Maybe be a bit more specific here, e.g. `Generate Newsletter Issue` or similar, as this is not about an issue(==problem) with the newsletter, but to open a github issue as a reminder. But this is very much nitpicking, so feel free to ignore.